lib/kanta_web/live/translations/translation_form_live/components/plural_translation_form/plural_translation_form.html.heex

```diff
@@ -1,16 +1,16 @@
 <div>
-  <div class="mb-4">
+  <div class="mb-6">
     <div>
       <div>
         <nav class="sm:hidden" aria-label="Back">
-          <a href="#" class="flex items-center text-sm font-medium text-slate-400 hover:text-slate-200">
+          <.link navigate={unverified_path(@socket, Kanta.Router, "/kanta/locales/#{@locale.id}/translations")} class="flex items-center text-sm font-medium text-slate-400 hover:text-slate-200">
```

Comment:
Why unverified path?
It's in the router so it should work with `~p"/kanta/locales/#{@locale.id}/translations"`. No?

https://github.com/curiosum-dev/kanta/blob/main/lib/kanta_web/router.ex#L46

Reply:
I have just copied it from `singular_translation_form` without going into too much detail. With `~p`, I've got an error: `** (RuntimeError) expected @endpoint to be set. For dynamic endpoint resolution, use path/2 instead.`. I think changing this to verified paths is out of the scope of this PR, as unverified paths are used all across the app. Let's leave it for a refactor.